cluster/examples/kubernetes/ceph/object-realm.yaml

```diff
@@ -8,3 +8,7 @@ kind: CephObjectRealm
 metadata:
   name: realm-a
   namespace: rook-ceph
+# This endpoint in this section needs is an endpoint from the master zone  in the master zone group of realm-a. See object-multisite.md for more details.
```

Comment:
I thought we were just adding examples in object-multisite.yaml, and no need for the individual object-realm, object-zone, and object-zonegroup.yaml. Why do we need the separate files? They all work together for a single multisite example, right?

Reply:
Yeah, all of these files are just concatenated together in the multisite example. I can remove all of these then.